src/modules/linagora.esn.graceperiod/frontend/js/app.run.js

```diff
@@ -1,8 +1,8 @@
-(function() {
+(function(angular) {
   'use strict';
 
   angular.module('linagora.esn.graceperiod')
     .run(function(gracePeriodLiveNotificationService) {
```

Comment:
I did not check everywhere but you need to `require` the file where `gracePeriodLiveNotificationService` is defined

Reply:
fixed